docs/Changelog.md

```diff
@@ -14102,6 +14102,63 @@ This version of the operator has been available since version 12 of the default
 <dd>Constrain index tensor to int64</dd>
 </dl>
 
+### <a name="MeanSquaredError-12"></a>**MeanSquaredError-12**</a>
+
+  Loss function that measures the
+  mean squared error (squared L2 norm) between each element in the 'scores'
+  and 'labels'.
+  
+  The loss can be described as:
```

Comment:
Before introducing this equation (output of loss function), we need to describe the shapes of `score` and `label` (inputs of loss function).